src/Redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { createStore } from "redux";
+import reducer from "../Redux/reducers/reducer";
+
+const reduxdevTool =
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__();
+const store = createStore(reducer, reduxdevTool);
```

Comment:
use redux saga here

Reply:
sir,i am having some issue with redux saga...i will change it later